pym/portage/package/ebuild/config.py

```diff
@@ -499,8 +505,10 @@ def __init__(self, clone=None, mycpv=None, config_profile_path=None,
 			#Loading Repositories
 			self["PORTAGE_CONFIGROOT"] = config_root
 			self["ROOT"] = target_root
+			self["SYSROOT"] = sysroot
 			self["EPREFIX"] = eprefix
 			self["EROOT"] = eroot
+			self["ESYSROOT"] = esysroot
```

Comment:
We need BROOT here too, right?

Reply:
Yep. Must have missed that when I reworked things. While we're here, could you explain what the `backup_changes` stuff below is for? Do I need to add BROOT there too?

Reply:
Yes, BROOT needs the `backup_changes` thing too. All that `backup_changes`  does is store the variable in self.backupenv, which causes it to persist through the mutations that occur when we call the `setcpv` method to configure settings for a specific package.